diff --git a/src/rswrapper.c b/src/rswrapper.c
index 620f0e0..d62ef1e 100644
--- a/src/rswrapper.c
+++ b/src/rswrapper.c
@@ -2,7 +2,11 @@
  * @file src/rswrapper.c
  * @brief Wrappers for nanors vectorization with different ISA options
  */
-
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <assert.h>
 // _FORTIY_SOURCE can cause some versions of GCC to try to inline
 // memset() with incompatible target options when compiling rs.c
 #ifdef _FORTIFY_SOURCE
\ No newline at end of file
